vendor/k8s.io/klog/v2/klog.go

```diff
@@ -1320,7 +1320,15 @@ func (v Verbose) Infof(format string, args ...interface{}) {
 // See the documentation of V for usage.
 func (v Verbose) InfoS(msg string, keysAndValues ...interface{}) {
 	if v.enabled {
-		logging.infoS(v.logr, msg, keysAndValues)
+		logging.infoS(v.logr, msg, keysAndValues...)
+	}
+}
+
+// Error is equivalent to the global Error function, guarded by the value of v.
+// See the documentation of V for usage.
+func (v Verbose) Error(err error, msg string, args ...interface{}) {
```

Comment:
should this be ErrorS? the signature doesn't match the global Error function as the godoc claims

Reply:
Yes, this looks like a bug in klog. We are upgrading to klog to pick up a bug fix https://github.com/kubernetes/klog/issues/165 We should be ok with this as currently we are not using klog.V(2).Error.

Reply:
> this looks like a bug in klog

and klog tagged a release with this? that means we can't change the signature until klog/v3, which I hope we never get to

Reply:
@dims proposed to add ErrorS and add comment to discourage people from using Error.
I don't think this is blocking. Are we ok to merge?